### DISCUSSION TITLE
Same cover for different albums

### DISCUSSION THREAD

**Discussion by wyllwyc** [navidrome/navidrome#3322] [Q&A] [answered]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [X] I'm using the latest version (your issue may have been fixed already)

### Version

latest

### Current Behavior

<img width="1280" alt="PixPin_2024-09-28_01-05-11" src="https://github.com/user-attachments/assets/c27d6106-e488-4dcc-b0e0-9a247ffe3d62">
All album covers become the same one

### Expected Behavior

All album covers display normally

### Steps To Reproduce

Use luci-app-dockerman to install

### Environment

```markdown
- OS:ImmortalWrt 23.05.3 
- Browser:Edge129.0.2792.52 on Windows11
- Client:
```

### How Navidrome is installed?

Docker

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by wyllwyc** [comment]
The album cover is correct.
![Screenshot_2024-09-28-01-20-07-996_cn aqzscn stream_music](https://github.com/user-attachments/assets/bc7402c4-82f8-48f4-a3dd-9c3f163531d8)
![Screenshot_2024-09-28-01-20-24-847_cn aqzscn stream_music](https://github.com/user-attachments/assets/60c532ee-d0fb-4e95-9962-8fe341554106)

**Comment by deluan** [comment]
How your library directories setup? Do you have multiple albums in the same folder? If yes, you may have a `cover.jpg` in that folder.

If that's not the case, logs will be needed:
1. Put this config in your `navidrome.toml`:
```
LogLevel = "info"
ImageCacheSize = "0"
[DevLogLevels]
"core/artwork" = "trace"
"utils/cache/" = "trace"
"core/ffmpeg" = "trace"
```
2. Restart Navidrome
3. Access the albums page
4. Send the logs from the moment you accessed the albums page til it was completely rendered.

**Comment by wyllwyc** [comment, marked as answer]
Thank you very much.There are two images in the music folder.Sorry for bothering you
